.agents/skills/nemoclaw-maintainer-analyze-ci-performance/scripts/analyze-recent-cli-timings.mts

```
@@ -0,0 +1,473 @@
+      minSamples: minimum,
+      runs,
+      slowTests,
+      slowFiles,
+    };
+  } finally {
+    const cleanup = await run("rm -rf -- " + quote(root), 30_000);
+    if (cleanup.exitCode !== 0)
+      throw new Error(
+        "Could not remove private temporary directory: " +
+          (await project(cleanup.stderr || cleanup.stdout, 1_000)),
+      );
```

Biome (2.5.8)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)